core/components/atoms/selectionCard/hooks/useMultiSelect.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+export function useMultiSelect() {
+  const [selectedItemList, setSelectedItemList] = React.useState<string[]>([]);
+
+  const updateSelection = (item: string) => {
+    let list = [...selectedItemList];
+
+    if (selectedItemList.includes(item)) {
+      list = selectedItemList.filter((cardItem: string) => item !== cardItem);
```

Comment:
Please use object to pass in value and implement selection based on that, if not possible use only one type for value that is object and put selection flag in the value to circulate through the hooks for selection toggle.